lib/hanami/application/view/context.rb

```diff
@@ -0,0 +1,95 @@
+# frozen_string_literal: true
+
+require "hanami/view"
+require "hanami/view/context"
+require_relative "../../errors"
+require_relative "../../slice_configurable"
+require_relative "slice_configured_context"
+
+module Hanami
+  class Application
+    class View < Hanami::View
+      # View context for views in Hanami applications.
+      #
+      # @api public
+      # @since 2.0.0
+      class Context < Hanami::View::Context
+        extend Hanami::SliceConfigurable
+
+        # @api private
+        def self.configure_for_slice(slice)
+          extend SliceConfiguredContext.new(slice)
+        end
+
+        # @see SliceConfiguredContext#define_new
+        def initialize(**kwargs)
+          defaults = {content: {}}
+
+          super(**kwargs, **defaults)
+        end
+
+        def inflector
+          _options.fetch(:inflector)
```

Comment:
@timriley Where `_options` are defined?

Reply:
@jodosha It's from `Hanami::View::Context` itself: https://github.com/hanami/view/blob/ec90f452facf349d15b536ea8b8f1c46cfd1c3a1/lib/hanami/view/context.rb#L57-L60

The reason why we have to capture the options like this is to allow the context to be reinitialized for each render_env. This is something that I'd like to address before hanami-view 2.0: it should be possible to keep a singular context object across the whole rendering life cycle if we use a dry-effects reader effect to give it access to the render_env, [which I wrote about here](https://timriley.info/writing/2020/10/06/open-source-status-update-september-2020/).